backup/queries_statistics.go

```diff
@@ -137,7 +145,7 @@ type TupleStatistic struct {
 	RelTuples float64
 }
 
-func GetTupleStatistics(connectionPool *dbconn.DBConn, tables []Table) map[uint32]TupleStatistic {
+func GetTupleStatisticsRows(connectionPool *dbconn.DBConn, tables []Table) (*sqlx.Rows, error) {
```

Comment:
This function is defined and used only in `backup` package. What do you think about changing it to `getTupleStatisticsRows` to forbid using in other packages?

And the same question for `GetAttributeStatisticsRows`.

Reply:
renamed